### PR TITLE
adding clingoLP section and examples

### PR DESCRIPTION
**Pull request by schellhorn**

### CODE REVIEW

**Comment by rkaminsk** [comment]
@sthiele can you have a look if this is okay? If so, I would merge it.

File: theory.tex [hunk 1]
```diff
@@ -436,15 +436,231 @@ \subsubsection{Modeling with Difference Constraints}
...
+%
+We may drop a condition if it is a tautology. 
+%
+Note that \clingoM{LP} supports further relations, viz. \code{>=}, \code{<}, \code{>}, \code{=} and \code{!=}.
+%
+Linear constraint atoms may occur in heads and bodies of rules. 
+%
+A formal introduction of \clingoM{LP} is given in \cite{jakaosscscwa17a}. 
+%
+The following description is conform with the \clingoM{LP} system available at \url{http:github.com/potassco/clingoLP}.
```
Comment (sthiele) [inline theory.tex:469]: Should this be?  https://github.com/potassco/clingoLP

Comment (schellhorn) [reply theory.tex:469]: Both work, but use the correct one: https://github.com/potassco/clingoLP

Comment (rkaminsk) [reply theory.tex:469]: Yes, always prefer https!

Comment (rkaminsk) [reply theory.tex:469]: I just took the liberty to fix it.

**Review by sthiele — CHANGES_REQUESTED** [review]

File: theory.tex [hunk 2]
```diff
@@ -436,15 +436,231 @@ \subsubsection{Modeling with Difference Constraints}
...
+\lstinputlisting{examples/lp1.lp}
+%
+Note that this program is analogously to the one of Example~\ref{ex:dl:simple}, but using real-valued constant \code{"4.2"}. 
+%
+Calling the system with the example produces following output:
+\marginlabel{
+  To inspect the output, invoke:\\
+  \code{\mbox{~}clingo clingoLP.py \textbackslash\\ 
+        \mbox{~~~}\attach{examples/lp1.lp}{lp1.lp}   0}\\
+}
```
[the remark this answers is not in the document]
Comment (rkaminsk) [reply theory.tex:522]: @sthiele you could also open a new PR. I think @schellhorn would just press the button anyway.

[the remark this answers is not in the document]
Comment (schellhorn) [reply theory.tex:522]: Yes Sven, you are right. Please go ahaed and adapt the command and the examples as needed. lp1.lp is not in examples as well. If changes are done, we may close the PR.

**Comment by rkaminsk** [comment]
Continued in #30.

**Pull request closed without merging**